openshift/README.md

```diff
@@ -0,0 +1,44 @@
+## How to use
```

Comment:
Put in top-level README so that we have all docs centraliesd?

(Wanted to introduce "Results" section for the two tests we'll do, but let's talk IRL...)